packages/devtools_app/lib/src/screens/inspector/diagnostics_node.dart

```diff
@@ -411,51 +415,51 @@ class RemoteDiagnosticsNode extends DiagnosticableTree {
   /// would return null. This will allow showing nested data for properties
   /// that don't show children by default in other debugging output but
   /// could.
-  bool get isDiagnosticableValue {
+  bool? get isDiagnosticableValue {
     return getBooleanMember('isDiagnosticableValue', false);
   }
 
-  String getStringMember(String memberName) {
+  String? getStringMember(String memberName) {
     return JsonUtils.getStringMember(json, memberName);
   }
 
   bool getBooleanMember(String memberName, bool defaultValue) {
     if (json[memberName] == null) {
       return defaultValue;
     }
-    return json[memberName];
+
+    return json[memberName] as bool;
   }
 
   DiagnosticLevel getLevelMember(
       String memberName, DiagnosticLevel defaultValue) {
-    final String value = json[memberName];
+    final String? value = json[memberName] as String?;
```

Comment:
remove LHS type. In general, it is not necessary to have both a type on the LHS and the RHS.

Reply:
done